### PR TITLE
Update code challenge.

### PR DESCRIPTION
**Pull request by aagm**
I have changed it a bit. I don't want to constrain them. Also, I am against giving them too much help unless they request it, it will show their ability to ask and their independence.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,27 +14,28 @@ meant to be an exercise to help us gauge where you are in terms of your current
...
-### Your Task:
-Use Python to perform a simple analysis of the data table from the following url:
-https://firms.modaps.eosdis.nasa.gov/active_fire/viirs/shapes/zips/VNP14IMGTDL_NRT_Global_7d.zip
-We would like you to analyse the data in relation to the areas of interest in the ares.geojson file.
-At minimum we would like to see at least one map and statistical chart.
+### The data
+Our main focus right now are supply chains, and how they affect the big challenges we face: climate change, water&food security and biodiversity lost. 
 
-### Hints and resources
+We would like you to use one of the datasets we are familiar with: [Mapspam](https://www.mapspam.info/data/)
```
Comment (tamaramegan) [inline README.md]: I would give all of them the same dataset, so we can have a common starting point (dataset +areas)

File: README.md [hunk 2]
```diff
@@ -13,27 +14,28 @@ meant to be an exercise to help us gauge where you are in terms of your current
...
+### The challenge
+
+We would like you to analyse the data in relation to the areas of interest in the [areas.geojson](./areas.geojson) file and contextualize it.
+
+At minimum we would like to see at least one map and a statistical chart.
 
-https://www.codecademy.com  (a nice git/github tutorial)
-https://software-carpentry.org/lessons/ (Open source lessons for Science-flavoured people)
+### The tools
+Use whatever ones you are comfortable with or you think are the best for the job.
```
Comment (tamaramegan) [inline README.md]: All candidates must know R or Python, I would keep that as it was in the previous one. If they decide to use ArcGIS for example, we may not be able to reproduce the analysis. 
Something like:
Use whatever ones you are comfortable with or you think are the best for the job, but keep in mind that the analysis should be reproducible and we will need to see some kind of script. We generally work with R or Python inside Jupyter notebooks, but other tools can work as well.

File: README.md [hunk 3]
```diff
@@ -13,27 +14,28 @@ meant to be an exercise to help us gauge where you are in terms of your current
...
+
+### How should I deliver the results?
+Add your results to your Github profile in a Jupyter notebook and be prepared to discuss them with us!
 
+The technical solution is not the only thing that we value, also your approach and explanations.
 
-Possible libraries you may want to consider: pandas, geopandas, fiona, rasterio, shapely, folium
+### How much time should I spend?
+As little as possible! We understand you probably have other duties so take your time.
```
Comment (tamaramegan) [inline README.md]: There is no time limit but we wouldn't want you to spend too much time working on this challenge, we understand you will probably have other things to do. 4-8 hours should allow you to explore the dataset and create the minimum visualizations (chart + map)

File: README.md [hunk 4]
```diff
@@ -13,27 +14,28 @@ meant to be an exercise to help us gauge where you are in terms of your current
...
 
-Metada for viirs
-https://earthdata.nasa.gov/earth-observation-data/near-real-time/firms/viirs-i-band-active-fire-data
+### What if I have questions?
+Emails us your questions at info@vizzuliaty.com
 
-Extra practica hint:
-If you have trouble setting-up your system, we reccomend using Anaconda Python.
-https://www.anaconda.com/distribution/
```
Comment (tamaramegan) [inline README.md:38]: I like to have some tips about were to go if they get stuck, maybe not the ones that were in the previous challenge but some guided information can help

**Review by tamaramegan — COMMENTED** [review]
I have left some comments, I think we can think a bit more about the challenge and what we expect to align before we send them out

File: README.md (hunk 2 above)
Comment (aagm) [reply README.md]: We explicitly say to use a jupyter notebook to share the analysis. I don't care if they executed a bash command in the notebook with gdal, with arcpy or with geopandas.

File: README.md (hunk 1 above)
Comment (aagm) [reply README.md]: Yes Mapspam is a huge dataset and for me is the same if they analyse soybean that wheat, as I say the data is similar if not the same and it gives them room enough to be creative.

File: README.md (hunk 4 above)
Comment (aagm) [reply README.md:38]: No, if they get stuck they can ask. Or demonstrate their independence and use of Google, so I will be against any tip in the challenge.

File: README.md (hunk 1 above)
Comment (aagm) [reply README.md]: Also it is a test to see if they understand the variables and the metadata

**Comment by fgassert** [comment]
I think its a good idea to align this with our supply chain work somewhat. 

In general, I would bias towards providing more information on what we're hoping for, as that will help us get more consistent results and avoid misunderstanding. e.g. MapSPAM is crop production statistics, so by analysis we're essentially asking for summary stats for these AOI? With the fires there was a real time component, so there could be trends and some sort of causal commentary. With crops they'll have to compare across technologies or crop types.

Most of these comments/edits otherwise are minor details I think and won't make a much of a difference either way. So, I think this is good to merge, but I would leave ownership/final edit to @tamaramegan as she is leading the hiring process.

**Review by fgassert — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tamaramegan) [reply README.md]: I've been looking at the data and for example there is barely any wheat in the areas of interest we are attaching. I'll propose to use soy and yams

**Pull request merged**